xunit-autolabeler-v2/ast_parser/python/source_parsers/test_data/webapp2_router_example.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################################################################
+# NOTE: This file is *sample data*, so it need only compile (not run!) #
```

Comment:
Nit: worth putting this note in other files (that I know _will_ run)?

Reply:
I'm not sure. I don't care much.